packages/node/src/lib/types.ts

```diff
@@ -0,0 +1,56 @@
+import { HTTPClient } from './http-client'
+
+export type OAuthSettings = {
```

Comment:
Since this is a public type, I think interface is best practice since it allows declaration merging (for lib consumers who want to augment for whatever reason)

Reply:
Done